pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -58,6 +59,10 @@ public static String getURL() {
     return getURL(getServer(), + getPort());
   }
 
+  public static String getAwsURL() {
+    return getURL().replace("jdbc:postgresql://", "jdbc:postgresql:aws://");
```

Comment:
Do we need to refer AWS protocol in community class?

Reply:
Not for community, I used it for AwsDriverTest.java but I can just make a private function for that test class.